src/sdk/config_fs.js

```diff
@@ -579,6 +582,25 @@ class ConfigFS {
         await native_fs_utils.folder_delete(account_dir_path, this.fs_context, undefined, true);
     }
 
+
+    /**
+     * _prepare_for_account_schema processes account data before writing it to the config dir and does the following -
+     * 1. encrypts its access keys
+     * 2. sets the used master key on the account
+     * 3. removes undefined properties, unwrap sensitive_strings and creation_data to string
+     * 4. checks account schema validation
+     * 5. and returns stringified data ready to be written to the config directory
```

Comment:
I think the text is not updated with the actual return value (missing `parsed_account_data`)